client/src/components/Ticketing/cart/Cart.tsx

```diff
@@ -44,8 +52,21 @@ const Cart = () => {
   const [targetItem, setTargetItem] = useState<number|null>(null);
   const [removeContext, setRemoveContext] = useState(RemoveContext.single);
   const [removeContextMessage, setRemoveContextMessage] = useState('');
+  const [discountText, setDiscountText] = useState<string|null>(null);
+  const [validDiscount, setValidDiscount] = useState(false);
+  const [discountClicked, setDiscountClicked] = useState(false);
+  const discount = useAppSelector(selectDiscount);
+  const total = totalReducer(subtotal, discount);
 
-  useEffect(() => console.log(subtotal), [subtotal]);
+  useEffect(() => {
+    console.log('Subtotal:', subtotal), [subtotal];
+    console.log('Total:', total), [total];
+    console.log('Current discount:', discount);
```

Comment:
These make the console really spammy whenever a user inputs anything, are these important or can we remove them?

Reply:
Yes, I had intended to take them out but forgot. The subtotal line was already in there but even it could go.